.buildkite/common.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Don't we remove this common part from `.buildkite/pipeline_pr.py`?

Reply:
That's the plan, but I want to do this in parts.